code/modules/jobs/job_types/enclave.dm

```diff
@@ -621,5 +613,12 @@
 /datum/job/enclave/encborg/after_spawn(mob/living/silicon/robot/R, mob/M)
 	. = ..()
 	ADD_TRAIT(R, TRAIT_TECHNOPHREAK, TRAIT_GENERIC)
+	ADD_TRAIT(R, TRAIT_ENCLAVE_CODES, src)
 	R.apply_pref_name("cyborg", M.client)
 	R.gender = NEUTER
+
+// Rappel stuff here
+/obj/machinery/abductor/console/enclave
+	name = "vertibird rappel console"
```

Comment:
Make this use a trait other than `TRAIT_ABDUCTOR_TRAINING`. You could make the needed trait a var instead, but don't give the Enclave abductor traits.